### DISCUSSION TITLE
MathJax macros don't seem to be working

### DISCUSSION THREAD

**Discussion by billhails** [community/community#56196] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,
According to the [MathJax](https://docs.mathjax.org/en/latest/input/tex/macros.html) documentation, you can define
macros that should be visible for the rest of the page, but when I try it:

$$
\def\e{{\mathtt{e}}}
\e
$$

it's not working. Are there plans to fix this, or a reason why it's disabled? (or have I just made a stupid mistake? :grin: )

**Comment by dpvc** [comment, marked as answer]
The GitHub configuration doesn't seem to include the `newcommand` extension that implements the `\def`, `\newcommand`, and related commands.  You are not doing anything wrong, they just haven't enabled the feature.  They also don't include the `require` extension, so you can't use `\require{newcommand}` to load the needed extension.  So it looks like you are out of luck, unfortunately.

**Reply by RoyiAvital** [reply]
Where is the configuration defined?

**Reply by dpvc** [reply]
You can view the `MathJax.config` object in the browser developer's console to see the configuration.  I'm not sure where it is in the code that they send to the browser, but that variable holds the details of the configuration.
